localstack/config.py

```diff
@@ -12,6 +12,9 @@
 
 TRUE_VALUES = ('1', 'true')
 
+# java options to Lambda
+
+JAVA_OPTS = os.environ.get('JAVA_OPTS', '').strip()
```

Comment:
Please rename this to `LAMBDA_JAVA_OPTS`, and add a short description to the README.